Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -62,28 +73,41 @@ public function flushNodeAggregate(
         ContentStreamId $contentStreamId,
         NodeAggregateId $nodeAggregateId
     ): void {
-        $tagsToFlush = [];
-
         $tagsToFlush[ContentCache::TAG_EVERYTHING] = 'which were tagged with "Everything".';
 
-        $this->registerChangeOnNodeIdentifier($contentRepository->id, $contentStreamId, $nodeAggregateId, $tagsToFlush);
+        $tagsToFlush = array_merge(
+            $this->collectTagsForChangeOnNodeAggregate($contentRepository, $contentStreamId, $nodeAggregateId),
+            $tagsToFlush
+        );
+
+        $this->flushTags($tagsToFlush);
+    }
+
+    /**
+     * @return array<string,string>
+     */
+    protected function collectTagsForChangeOnNodeAggregate(
```

Comment:
why protected?

Reply:
You would prefer private?

Reply:
If you ask me like that: Yes, I prefer final classes and private fields :) 
But in this case I ask because the other `collect*` methods are already private

Reply:
Is changed.